dienst2/settings.py

```diff
@@ -37,6 +37,10 @@
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
+LOCALE_PATHS = (
+    os.path.join(os.path.dirname(__file__), 'locale'),
```

Comment:
Isn't this default behaviour?
https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#how-django-discovers-translations